Makefile

```diff
@@ -87,6 +87,10 @@ endif
 
 GO_TEST_GINKGO_ARGS ?= -test.v -ginkgo.v -ginkgo.failFast
 
+ifeq ($(DOCKERCMD),)
+DOCKERCMD = podman
+endif
```

Comment:
Why not use the same method used for other variables?

    DOCKERCMD ?= podman

See https://www.gnu.org/software/make/manual/make.html#Setting

Reply:
Thanks, made the update now.